tests/run-make/no-mark-noalias/rmake.rs

```diff
@@ -0,0 +1,14 @@
+// In the rustc_trans crate, references returned by a Frozen pointer type
+// could mark them as "noalias", which caused miscompilation errors.
```

Comment:
I feel this should be a codegen test if we're trying to check for `noalias` metadata?